sdks/python/apache_beam/io/tfrecordio.py

```diff
@@ -57,7 +57,7 @@ def _default_crc32c_fn(value):
 
     if not _default_crc32c_fn.fn:
       _LOGGER.warning(
-          'Couldn\'t find python-snappy so the implementation of '
+          'Couldn\'t find python-snappy (<0.7) so the implementation of '
```

Comment:
+0 (I don't think this is necessary.) I think requirements should be controlled outside of code (e.g. via extras), if that is necessary. 

if you decide to merge, you could write: 

```suggestion
          'Couldn\'t find python-snappy<0.7 so the implementation of '
```

Reply:
well, this one is tricky since snappy (>=0.7) completely removes `crc32c`. Better call this out clearly.